test/checkImageSize.js

```diff
@@ -0,0 +1,87 @@
+var fixImage = require("./fixImageSize.js");
+const sizeOf = require('image-size');
+const fs = require('fs');
+const { exit } = require("process");
+
+function getFileSizeMB(fileStats){
+    return (fileStats.size) / (1024*1024);
+}
+
+//Check if image is larger than 1MB
+function isFileSizeBig(fileStats){ 
+    return getFileSizeMB(fileStats) > 1;
+}
+
+function getImageWidth(imageSize){
+    return imageSize.width;
+}
+
+function getImageHeight(imageSize){
+    return imageSize.height;
+}
```

Comment:
These two functions can probably just be replaced by accessing the property; I don't think they add anything to encapsulation.